src/main/java/seedu/apollo/ui/Parser.java

```diff
@@ -136,10 +146,20 @@ private static Command parseCommand(String[] split, int size, ModuleList moduleD
             return new ExitCommand();
 
         case COMMAND_HELP_WORD:
-            if (!isOneWord(split)) {
+            if (!isOneWord(split) && (!split[1].equals("todo") &&!split[1].equals("deadline")&&!split[1].equals("event")
+                    &&!split[1].equals("mark")&&!split[1].equals("unmark")&&!split[1].equals("delete")
+                    &&!split[1].equals("find") &&!split[1].equals("date") &&!split[1].equals("addmod")
+                    &&!split[1].equals("delmod")&&!split[1].equals("week") &&!split[1].equals("showmod")
+                    &&!split[1].equals("listmod") &&!split[1].equals("list"))) {
```

Comment:
There is a risk of this becoming extremely long, is there a way to refactor this?

Reply:
okay i will make another boolean method that returns true if its not any of the known words

Reply:
Suggest using a set of strings:
```
Set<String> allowedCommands = new HashSet<>(Arrays.asList("todo", "deadline", "event", "mark", "unmark", "delete", "find", "date", "addmod", "delmod", "week", "showmod", "listmod", "list"...));

if (!isOneWord(split) && !allowedCommands.contains(split[1])){
// other code here
}

```

Reply:
Or you can utilise your `chooseHelpCommand()` method to reduce redundant code.